src/reporters/cli.js

```diff
@@ -0,0 +1,80 @@
+const rightpad = require('right-pad')
+const figures = require('figures')
+const bytes = require('bytes')
+const plur = require('plur')
+
+const colors = require('../utils/colors')
+
+function report(results) {
+  const files = results.files
+
+  const maxFileLength = getMaxFileLenght(files)
+
+  files.forEach(function(row) {
+    printBlockHeader(row)
+
+    row.filesMatched.forEach(function(file) {
+      printRow(file, row, maxFileLength)
+    })
+  })
+
+  printSummary(results.counter)
+}
+
+module.exports = { report }
+
+function getMaxFileLenght(files) {
```

Comment:
Looks like a minor typo here:

```suggestion
function getMaxFileLength(files) {
```

Reply:
You also gotta fix it at the site of usage 😄